client/src/leap/soledad/client/adbapi.py

```diff
@@ -217,6 +217,9 @@ def _runU1DBQuery(self, trans, meth, *args, **kw):
         """
         meth = getattr(trans, meth)
         return meth(*args, **kw)
+        # XXX should return a fetchall?
```

Comment:
I've been taking a look on how to convert the whole u1db sqlite plugin to twisted async, and the way i visualize it, it would be a whole lot of work. Not sure if this is strictly related with your commen above.